underfs/s3a/src/main/java/alluxio/underfs/s3a/S3AUnderFileSystem.java

```diff
@@ -218,7 +218,7 @@ public static S3AUnderFileSystem createInstance(AlluxioURI uri,
             numTransferThreads).create();
 
     TransferManager transferManager = TransferManagerBuilder.standard()
-        .withS3Client(createAmazonS3(credentials, clientConf, endpointConfiguration, conf))
+        .withS3Client(amazonS3Client)
```

Comment:
<img width="1806" alt="image" src="https://user-images.githubusercontent.com/39251482/171776522-44144434-baef-4636-b626-4737de1e4060.png">
 
The original design was using two awss3client, any idea the reason to have two?

Reply:
https://github.com/Alluxio/alluxio/commit/24cd14d5e003c8df1b8ba8685d2738ab0645c8d8

Reply:
I changed to use two, but highly likely an typo from my PR https://github.com/LuQQiu/alluxio/commit/36a4d5c93c2756bcaae5c76cc295a5442d6abad8
that PR does not need to change to use two separate clients

Reply:
The original design was using two awss3client 《-- it's not the original design but changed by me....